### DISCUSSION TITLE
Is it possible to show cy.log() in the module api result?

### DISCUSSION THREAD

**Discussion by davidjudilla** [cypress-io/cypress#16598] [Questions and Help]
I'm trying to run a test via the module api
```
    const result = await cypress
      .run({
        // the path is relative to the current working directory
        spec: './cypress/integration/accountCreation.int.spec.js',
      });
```
And inside the test there's a cy.log()
```
  cy.log(`User email: ${email}`)
```

I thought I would be able to see the log inside of `tests.body` from `results`, but sadly couldn't.
![Screen Shot 2021-05-19 at 3 17 48 PM](https://user-images.githubusercontent.com/11013674/118892259-79fac100-b8b5-11eb-86a1-ae349af0ff66.png)

Anyone know if it's possible to get a cy.log() or a console.log() to show up in the module api results?

**Comment by jennifer-shehane** [comment]
This should show up. Can you provide the full test code that you're running in your `test` test? Because it looks like it's more code than just `cy.log()` since there's `expect(true).to.be.false`

**Reply by davidjudilla** [reply]
Thanks for responding! The screenshot I posted was from the module api docs, just to make sure it was clear what part of the result was talking about. You're right there is more to it, here's my test code

```js
  // accountCreation.int.spec.js

  it('should create a user', () => {
    cy.createUser();
  });
```

```js
// commands.js

Cypress.Commands.add('createUser', () => {
  const email = `test@email.com`;
  const password = 'str0ngPassword1';

  cy.request({
    url: `/registerUser`,
    qs: {
      email,
      password,
    },
  }).as('registerUser');

  cy.get('@registerUser').then((response) => {
    assert.equal(response.status, 200);
  });

  cy.log(`User email: ${email}`)
  cy.log(`User password: ${password}`);
});
```

But this is the body I get from the module api test result

```js
"tests":[
            {
               "title":[
                  "account creation",
                  "should create a user"
               ],
               "state":"passed",
               "body":"() => {\n    cy.createUser();\n  }",
               "displayError":null,
```

It seems like `body` is just the content of the `it()` statement, so trying to figure out how to get those logs in the `createUser` command to show up in the result

**Reply by jennifer-shehane** [reply]
Yah, the `body` will only be the actual body of the test, in this case it only contains `cy.createUser()`, it wont evaluate every function of the test body and save the function definitions. 

It would probably be helpful to understand what your goal is in all of this. What's your end goal?

**Reply by davidjudilla** [reply]
The end goal would be to have the data from `cy.log()` show up somewhere in the module api run results. It's starting to look like I might need to make some sort of plugin or util to help support that
